examples/bootstrap/bootstrap.yaml

```diff
@@ -1,6 +1,6 @@
 resources:
 - name: expandybird
-  type: https://raw.githubusercontent.com/kubernetes/deployment-manager/master/templates/replicatedservice/v1/replicatedservice.py
+  type: github.com/kubernetes/application-dm-templates/common/replicatedservice:v1
```

Comment:
Note that until I get my changes in (to support the short URL types), this will not work. This should land either today (depending on how the wifi on the plane is), or tomorrow.